### ISSUE TITLE
multi ip cache

### ISSUE THREAD

**Issue by webconfss** [3proxy/3proxy#775]
hi.
now when authcache  ip enabled its just cache one ip at time
is it possible to add more ip to cache? example  authcache ip make cache 2 ip or more

**Comment by z3APA3A** [comment]
How multiple IPs are expected to be used?

**Comment by webconfss** [comment]
for multi login from one username can connect at same time.
in squid use this command can limit number of ip to cache
example for one ip:
acl ipmax max_user_ip -s 1
example for two ip:
acl ipmax max_user_ip -s 2

**Comment by z3APA3A** [comment]
Do you mean `authcache` with `limit`?

**Comment by webconfss** [comment]
yes but it limit to single ip address but i need to authcache  limit two ip address

**Comment by z3APA3A** [comment]
`connlim` allows to limit connections
